scripts/formater.pl

```diff
@@ -95,7 +95,7 @@
 
     my $pid;
     if($pid=fork()){
-	#do parrent staf
+	#do parent staf
```

Comment:
The spell-check.sh script will not catch every spelling error.
That is to be expected.
In this case, "staf" is not in our white list so we are not excluding it on purpose.
It must simply that it is not listed as a misspelling in the default dictionary.
So one way to handle this would be to create our own customized dictionary.
But that is beyond the scope of this PR.